tests-utils/src/main/scala/tech/beshu/ror/utils/containers/EsClusterContainer.scala

```diff
@@ -179,7 +179,7 @@ object EsVersion {
 
 sealed trait SecurityType
 object SecurityType {
-  final case class RorWithXpackSecurity(attributes: ReadonlyRestWithEnabledXpackSecurityPlugin.Config.Attributes) extends SecurityType
+  final case class RorWithXpackSecurity(attributes: ReadonlyRestWithEnabledXpackSecurityPlugin.Config.Attributes, performInstallation: Boolean = true) extends SecurityType
```

Comment:
IMO we should have this flag for `RorSecurity` too. It's (almost) always better to be symmetrical.

Reply:
I limited the usages and visibility of `performPatching` to only 2 places: 
- EsContainerWithRorAndXpackSecurity (where it is created with default settings - with patching enabled) 
- TestEsContainerManager for ror-tools tests - where it is created with custom settings

So it is no longer visible here and generally in utils used for standard integration tests.